DeepSpeech/Dockerfile.train

```diff
@@ -127,7 +131,7 @@ RUN git clone https://github.com/$kenlm_repo.git && cd kenlm && git checkout $ke
     && mkdir -p build \
     && cd build \
     && EIGEN3_ROOT=$HOMEDIR/eigen-eigen-07105f7124f9 cmake .. \
-    && make -j
+    && make -j $(nproc)
```

Comment:
as much as I recall, `-j` alone does the same

Reply:
Ok thanks! I'll remove it then.